### ISSUE TITLE
Issue with full sync

### ISSUE THREAD

**Issue by RonilPatel7** [Arthur-Milchior/anki-window-open-on-sync#3]
I have noticed an issue with "full one way sync"; the add on does not allow full sync to proceed; it gets stuck with a dialog box which says "processing"

This issue can be replicated by going to Preferences and clicking "__on next sync, force changes in one direction_" & is further demonstrated in this video_

https://www.dropbox.com/s/c9s349qmtjjako3/Issue%20-%20Sync%20Keep%20Window%20Open.mp4?dl=0

Any advice or guidance would be greatly appreciated

**Comment by Arthur-Milchior** [comment]
I'll need to investigate. I've not enough time right now.

**Comment by Arthur-Milchior** [comment]
Of course, there is an easy solution, restart anki without the add-on. Since full sync already close the window, that does not defeat the initial purpose of the add-on

**Comment by RonilPatel7** [comment]
yes you are correct; that easy solution does work; perhaps, we should update the documentation on ankiweb to indicate to people that it will not work when full sync is necessary so that people are aware?

**Comment by Arthur-Milchior** [comment]
I edited to state to just restart without this add-on if sync fail. That's not the expected behaviour and I hope I'll fix it soon. Normally, in case of full sync, it goes back to anki's built-in syncing method, it should not just fail. I've too little time to investigate, and sadly, this is one of the ugliest add-on I've wrote, so there is no realistic way to try to fix it without spending hours on it

**Comment by RonilPatel7** [comment]
Thank you for taking the time to look into it; I hope that you job is going well!
